packages/math-input/src/components/keypad/keypad.tsx

```diff
@@ -130,108 +80,73 @@ export default function Keypad({extraKeys = [], ...props}: Props) {
-                            <FractionsPage
-                                onClickKey={onClickKey}
-                                cursorContext={cursorContext}
-                            />
-                        )}
-                        {selectedPage === "Numbers" && (
-                            <NumbersPage
-                                onClickKey={onClickKey}
-                                scientific={scientific}
-                            />
-                        )}
-                        {selectedPage === "Extras" && (
-                            <ExtrasPage
-                                onClickKey={onClickKey}
-                                extraKeys={extraKeys}
-                            />
-                        )}
-                        {selectedPage === "Operators" && (
-                            <OperatorsPage
-                                onClickKey={onClickKey}
-                                preAlgebra={preAlgebra}
-                                logarithms={logarithms}
-                                basicRelations={basicRelations}
-                                advancedRelations={advancedRelations}
-                            />
-                        )}
-                        {selectedPage === "Geometry" && (
-                            <GeometryPage onClickKey={onClickKey} />
-                        )}
-                        {!fractionsOnly && (
-                            <SharedKeys
-                                onClickKey={onClickKey}
-                                cursorContext={cursorContext}
-                                convertDotToTimes={convertDotToTimes}
-                                divisionKey={divisionKey}
-                                selectedPage={selectedPage}
-                            />
-                        )}
+                        <IconButton
+                            icon={xBold}
+                            kind="tertiary"
+                            aria-label="Dismiss"
+                            onClick={() => onClickKey("DISMISS")}
+                            size="xsmall"
+                            tabIndex={0}
+                            style={{
+                                color: semanticColor.core.foreground.neutral
+                                    .default,
+                            }}
+                        />
                     </View>
-                    {expandedView && <NavigationPad onClickKey={onClickKey} />}
-                </View>
+                )}
+                <Tabs
+                    aria-label="Keypad"
+                    tabs={availableTabs}
+                    selectedTabId={selectedPage}
+                    onTabSelected={(newSelectedPage: string) => {
+                        setSelectedPage(newSelectedPage as KeypadPageType);
+                    }}
+                    styles={{
+                        tab: {
```

Comment:
Can we move these styles out of the JSX rendering? If we move them out of the component it will make the JSX easier to read and we'll get an added bonus in that we won't have to redeclare them every time we rerender.